templates/web/shropshire/index-steps.html

```diff
@@ -0,0 +1,18 @@
+<h2>[% loc('How to report a problem') %]</h2>
+
+<ol class="big-numbers">
+    <li>[% question %]</li>
+    <li>Plot the location on the map</li>
+    <li>[% loc('Enter details of the problem') %]</li>
+    [% IF c.cobrand.is_council %]
```

Comment:
It is.

Reply:
(so this IF can be removed, I meant)

Reply:
This fix is now in master as was merged from another branch